.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/go:1.19
+      - image: cimg/go:1.21
```

Comment:
Thanks! But we've already stopped using circleci so you could just remove this out of this pull request!

Reply:
File deleted